include/boost/url/detail/impl/normalize.ipp

```diff
@@ -20,119 +20,119 @@ namespace boost {
 namespace urls {
 namespace detail {
 
+void
+pop_encoded_front(
```

Comment:
This was just extracted from `compare_encoded` so it could be reused. Same code here.